src/utils.ts

```diff
@@ -478,6 +483,18 @@ export function replaceWebRootInSourceMapPathOverridesEntry(webRoot: string, ent
     return entry;
 }
 
+export function addEntrypointIfNeeded(sourcePath: string, defaultEntrypoint: string): string {
+    try {
```

Comment:
What in this codeblock causes an exception we can't anticipated? It's not clear to me why we need a try/catch if we aren't logging an exception.

Reply:
So, the new URL() call will throw if the passed in string isn't a valid url. I can't think of a case where that would be, but I added the try catch just to be safe.

I'd be open to letting it throw throw and then catching in the devtoolsPanel and reporting an error instead of attempting to open the file. I'll go ahead and make that change